compliance_checker/ioos.py

```diff
@@ -157,23 +189,175 @@ def check_variable_units(self, ds):
 
         return Result(BaseCheck.MEDIUM, (count, len(ds.variables)), 'Variable Units', msgs)
 
-    def check_altitude_units(self, ds):
-        """
-        If there's a variable named z, it must have units.
 
-        @TODO: this is duplicated with check_variable_units
-        """
-        if 'z' in ds.variables:
-            msgs = []
-            val = 'units' in ds.variables['z'].ncattrs()
-            if not val:
-                msgs.append("Variable 'z' has no units attr")
-            return Result(BaseCheck.LOW, val, 'Altitude Units', msgs)
+class IOOS1_1Check(IOOSNCCheck):
+    '''
+    Compliance checker implementation of IOOS Metadata Profile for NetCDF, Version 1.1
 
-        return Result(BaseCheck.LOW, (0, 0), 'Altitude Units', ["Dataset has no 'z' variable"])
+    Related links:
+    https://ioos.github.io/ioos-netcdf/ioos-netcdf-metadata-description-v1-1.html#ioos-netcdf-metadata-profile-attributes
+    https://github.com/ioos/compliance-checker/issues/69
+    https://github.com/ioos/compliance-checker/issues/358
+    '''
+    _cc_spec_version = '1.1'
+    _cc_description = 'IOOS Metadata Profile for NetCDF, Version 1.1'
+    _cc_url = 'https://ioos.github.io/ioos-netcdf/ioos-netcdf-metadata-description-v1-1.html#ioos-netcdf-metadata-profile-attributes'
+    register_checker = True
+
+    def __init__(self):
+        super(IOOS1_1Check, self).__init__()
```

Comment:
I would strongly caution against using `super` and instead using 

```python
IOOS1_1Check.__init__(self)
```

I don't remember if this is the right reference as to why but it might link to it.
https://fuhm.net/super-harmful/

All I remember is that `super` bit us really really hard in the past.

Reply:
I'll check it out, I'm wondering if I need that line at all